ui/src/layouts/template.tsx

```diff
@@ -106,6 +116,9 @@ export function RenderedTemplate(props: any) {
                 readOnly={true}
                 name="RENDERED_TEMPLATE"
                 editorProps={{ $blockScrolling: true }}
+                setOptions={{
+                    useWorker: false,
```

Comment:
What is this option?

Reply:
This option is to make sure that any heavy process can work without affecting the UI. In our case it is the linting of JSON.